semeru-ce-11-rhel8.yaml

```diff
@@ -0,0 +1,89 @@
+# This is an Image descriptor for Cekit
+# use the following extra override to build:
+# --overrides "{ 'osbs': { 'configuration': { 'container': { 'platforms': { 'only': [ 's390x' ] }, 'compose': { 'signing_intent': 'release', 'packages': [ 'ibm-semeru-open-11-jdk' ], 'inherit': true } } } } }"
+
+schema_version: 1
+
+from: "registry.redhat.io/ubi8/ubi-minimal"
+name: &name "openj9/openj9-11-rhel8"
+version: &version "1.9"
+description: "Source To Image (S2I) image for Red Hat OpenShift providing IBM Semeru Runtime 11"
+
+labels:
+- name: "io.k8s.description"
+  value: "Platform for building and running plain Java applications (fat-jar and flat classpath)"
+- name: "io.k8s.display-name"
+  value: "Java Applications"
+- name: "io.openshift.tags"
+  value: "builder,java"
+- name: "maintainer"
+  value: "Red Hat OpenJDK <openjdk@redhat.com>"
+- name: "com.redhat.component"
+  value: "openj9-11-rhel8-container"
+- name: "usage"
+  value: "https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/html/red_hat_java_s2i_for_openshift/"
+
+envs:
+- name: PATH
+  value: $PATH:"/usr/local/s2i"
+- name: "JBOSS_IMAGE_NAME"
+  value: *name
+- name: "JBOSS_IMAGE_VERSION"
+  value: *version
+
+ports:
+- value: 8080
+- value: 8443
+
+packages:
+  manager: microdnf
+  content_sets:
+    x86_64:
+    - rhel-8-for-x86_64-baseos-rpms
+    - rhel-8-for-x86_64-appstream-rpms
```

Comment:
Why content sets for `x86_64` and `aarch64`? I thought this would be ppc64le and s390x only.